From 3575e84872c2714919085e7a8a0c386fadb438dd Mon Sep 17 00:00:00 2001
From: rofl0r <retnyg@gmx.net>
Date: Mon, 20 Jan 2014 21:31:34 +0100
Subject: [PATCH] remove inclusion of sysinfo.h in kernel.h

the declaration of struct sysinfo clashes with userspace.
it's not quite clear why that header was included from kernel.h,
as none of its functionality is needed.
---
 generic/include/linux/kernel.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/generic/include/linux/kernel.h b/generic/include/linux/kernel.h
index 642d1e9..9fe3cff 100644
--- a/generic/include/linux/kernel.h
+++ b/generic/include/linux/kernel.h
@@ -1,8 +1,6 @@
 #ifndef _LINUX_KERNEL_H
 #define _LINUX_KERNEL_H
 
-#include <linux/sysinfo.h>
-
 /*
  * 'kernel.h' contains some often-used function prototypes etc
  */
-- 
2.20.1

